Content.Server/Administration/Logs/Converters/EntityStringRepresentationConverter.cs

```diff
@@ -19,6 +27,43 @@ public override void Write(Utf8JsonWriter writer, EntityStringRepresentation val
         if (value.Session != null)
         {
             writer.WriteString("player", value.Session.UserId.UserId);
+
+            if (_adminManager.IsAdmin(value.Uid))
+            {
+                writer.WriteBoolean("admin", true);
```

Comment:
Remove one indent level as we don't need to ensure the session is not null now since we use a method requiring an EntityUid instead

Reply:
well, that’s probably fair, but it just doesn’t make any sense to do that check if you don’t pass the session in the method, Idk

Reply:
> At most, the admin check can just be moved out of the session check

Yeah, that’s what I meant

Reply:
I don't think the readability improvement warrants removing the null check. You could maybe remove the curly braces from ifs with just one line in their body to reduce nesting but that's a very subjective code style choice and we don't have that specified in our code style guidelines